orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/pipeline/BakeStage.groovy

```diff
@@ -174,7 +174,7 @@ class BakeStage implements StageDefinitionBuilder {
 
       if (failOnImageNameMismatchEnabled()) {
         // find distinct image names in bake stages that are actually related to the stage passed into the task
-        List<String> distinctImageNames = relatedBakeStages
+        List distinctImageNames = relatedBakeStages
```

Comment:
Can't this be
```suggestion
        List<StageExecution> distinctImageNames = relatedBakeStages
```
or at least `List<Object>` ?  I'd like to avoid using a raw type.

Reply:
Since this return list of object, I will make it as `List<Object>`.

Reply:
Done